### ISSUE TITLE
Firefox Console Error: The Components object is deprecated. It will soon be removed.

### ISSUE THREAD

**Issue by jplwood** [reactjs/React.NET#1101]
### Steps to reproduce
Open ReactJS.Net project in firefox, look at the console.
-------

Console error in firefox related to use of non-standard global object, which doesn't even seem to be necessary. As far as I can tell, the `Components` global that ReactJS.NET has you set up, doesn't seem to be using that particular global object for the functionality that seems to be deprecated. This is likely just side-effect of an arbitrary name. 

**Error:**
`The Components object is deprecated. It will soon be removed.`

**about it:** https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XPCOM/Language_Bindings/Components_object

I was able to suppress this error and have everything still work by renaming the global components variable (e.g., `MyComponents`).

Unless I'm misunderstanding

**Comment by dustinsoftware** [comment]
Yeah it should still work fine but you’re right, we should update the
template and docs to use another name for the global. My maybe
ReactComponents. Feel free to PR...
